### DISCUSSION TITLE
aiohttp<3.8.0 header Injection vulnerability

### DISCUSSION THREAD

**Discussion by Paprikar** [Rapptz/discord.py#8438] [General]
The vulnerability [has been fixed](https://github.com/aio-libs/aiohttp/issues/4818) ([PR](https://github.com/aio-libs/aiohttp/commit/3cae27ac9232b5b86b8837a9f540b8fc574489a0)) in version 3.8.0.
Is it worth opening an issue about it?

**Comment by Rapptz** [comment]
The library already uses the newest version lower than 4.0 now.
